src/bootlint.js

```diff
@@ -162,6 +164,51 @@ var LocationIndex = _location.LocationIndex;
         return runs;
     }
 
+    /**
+     * This function retrieves the browser window object and sets it to a variable.
```

Comment:
"This function returns the browser window object, or null if this is not running in a browser environment. "